### PR TITLE
Add support for java services in affectedModuleDetector plugin

### PR DESCRIPTION
**Pull request by tinder-ravitejabonagiri**
Addressing the issue in https://github.com/dropbox/AffectedModuleDetector/issues/70

### CODE REVIEW

File: affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorPlugin.kt [hunk 1]
```diff
@@ -74,9 +74,9 @@ class AffectedModuleDetectorPlugin : Plugin<Project> {
 
         rootProject.subprojects { project ->
             project.afterEvaluate {
-                val pluginIds = setOf("com.android.application", "com.android.library", "java-library", "kotlin")
+                val pluginIds = setOf("com.android.application", "com.android.library", "java", "kotlin")
```
Comment (chris-mitchell) [inline affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorPlugin.kt:77]: Thanks for the contribution!

Is there a reason we removed `java-library` from the `setOf`? Shouldn't we include both `java-library` and `java` ?

Comment (tinder-ravitejabonagiri) [reply affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorPlugin.kt:77]: `java-library` plugin applies `java` plugin. 
https://github.com/gradle/gradle/blob/master/subprojects/plugins/src/main/java/org/gradle/api/plugins/JavaLibraryPlugin.java#L51

Comment (chris-mitchell) [reply affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorPlugin.kt:77]: Thanks for clarifying.   Once the CLA is signed, we can merge this.

**Comment by joshafeinberg** [comment]
Closing this PR as the CLA has not been signed. This will be resolved in #134

**Pull request closed without merging**